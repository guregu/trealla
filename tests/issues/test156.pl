:-initialization(main).

main :-
	?=(a,a), writeln(here3),
	unifiable(a,a,X2), writeln(here2),
	unifiable(a,A,X1), writeln(here1),
	true.
