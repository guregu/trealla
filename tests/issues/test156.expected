here3
here2
here1
